.gitignore

```diff
@@ -1,6 +1,9 @@
 # Created by https://www.toptal.com/developers/gitignore/api/windows,macos,linux,python,react,pycharm,emacs,vim,visualstudio,visualstudiocode
 # Edit at https://www.toptal.com/developers/gitignore?templates=windows,macos,linux,python,react,pycharm,emacs,vim,visualstudio,visualstudiocode
 
+# envs
+venv*/
```

Comment:
What's the purpose? Seems like something that should not land.

Reply:
this is pretty standard in gitignores, used to ignore my dev env